pyne/mcnp.py

```diff
@@ -2285,3 +2297,24 @@ def _mesh_to_mat_cards(mesh, divs, frac_type):
         mat_cards += mesh.mats[i].mcnp(frac_type=frac_type)
 
     return mat_cards
+
+def _check_tag_names(tag_names):
+    """Make sure tag_names is an iterable of 4 strings."""
+    # check iterable
+    if check_iterable(tag_names):
```

Comment:
This might be simpler as separate if statements that each raise an exception upon failure:

```
if not check_iteraable(tag_names):
    raise ValueError("....")

if len(tag_names) != 4:
    raise ValueError("....")

````
etc

Reply:
Thanks. Suggestion adopted in https://github.com/pyne/pyne/pull/1199/commits/f4b6dccff3290e3d0bb6e6a646e029199c95c759.